kolena/workflow/visualization/_activation_map.py

```diff
@@ -70,6 +70,17 @@ def blue(self, intensity: np.uint8) -> float:
 
 
 def colorize_activation_map(activation_map: np.ndarray, fade_low_activation: bool = True) -> np.ndarray:
+    """
+    Applies the [MATLAB "Jet" colormap](http://blogs.mathworks.com/images/loren/73/colormapManip_14.png) to the
+    activation map.
+
+    :param np.ndarray activation_map: A 2D numpy array, shaped (h, w) or (h, w, 1), of the activation map in `np.uint8`
+        or `float` ranging [0, 1].
```

Comment:
Does `[0, 1]` only apply to float type inputs? Are floats of any precision accepted?

Reply:
Yes and yes